doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,71 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`_, it is possible to convert the app to an executable by taking a few precautions.
+Because it is necessary to have the VERSION of PyMAPDL for the app to work, the file need to be added next to our executable.
+
+Start by generating the `specification file <https://pyinstaller.org/en/stable/spec-files.html>`_ for the app.
+
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
+
+PyInstaller provides two modes for generating executables:
+
+- ``onedir`` (default): This mode generates a folder that includes the executable file along with its dependencies.
+- ``onefile``: In this mode, PyInstaller generates a single executable file.
+
+To generate the executable in "onefile" mode, include the argument ``--onefile`` in the command.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py --onefile
+
+Then, add the link to the ``VERSION`` file from the PyMAPDL package in ``cli_rotor.spec``
+
+.. code-block:: python
+   :emphasize-lines: 1-9,15
+
+   import os
+   import importlib
+
+   proot = os.path.dirname(importlib.import_module("ansys.api.mapdl").__file__)
```

Comment:
@RobPasMue It's weird that Vale is flagging issues here in a code block. I know Vale can be turned on and off before this block, but why would it have to be? Is there some formatting error or something that I'm missing?

Reply:
Outdated.